src/core/content/util.js

```diff
@@ -110,6 +110,16 @@ const Util = {
 		return { artist, track };
 	},
 
+	/**
+	 * Remove record side from track title.
+	 * @param {String} str String contains track title
+	 * @return {String} String contains track title without record side
+	 */
+	removeRecordSide(str) {
```

Comment:
Could we include a test case for this?

See https://github.com/web-scrobbler/w`eb-scrobbler/blob/3ca9d78037931c8d5517d08fde3b1f5d0330036d/tests/content/util.js#L23 How this is setup for `splitArtistTrack